src/app/app-routing.module.ts

```diff
@@ -21,6 +21,11 @@ const appRoutes: Routes = [
     loadChildren: () =>
       import('src/app/auth/auth.module').then((m) => m.AuthModule),
   },
+  {
+    path: 'profile',
+    loadChildren: () =>
+      import('src/app/profile/profile.module').then((m) => m.ProfileModule),
+  },
```

Comment:
You can simplify using `loadChildren: './profile/profile.module#ProfileModule'`

Reply:
I used it before, and changed it because in this case I got errors like Error: Cannot find module